### ISSUE TITLE
The auto extension was written for Europe and is unsuited for the US market 

### ISSUE THREAD

**Issue by JoeUX** [schemaorg/schemaorg#1549]
The auto extension is entirely situated in an EU context. The terms, units, and legal assumptions are all European. Most strikingly, there's no provision for _a car that a person owns_, which is extremely common in the US. I'll revisit that in a minute.

First, some examples of proprietary terms are trailerweight and ACRIS, and the units used for all the types. 

ACRIS is an EU protocol, if you will, for car rental companies.

Americans would say _towing capacity_ instead of _trailerweight_. And neither of the units of mass offered in any of these types is used in the US. We use _pounds_, but would never abbreviate pounds as _LBR_. The ANSI or NIST abbreviation is _lbs_. 

And the legal stuff about special use of the vehicle is centered on European law.

Like I said, I was struck that there's no type for just a car that someone owns. There's also no provision for the types of vehicles Americans drive. These are the four types in the schema:

- BusOrCoach
- CarUsageType
- Motorcycle
- MotorizedBicycle

CarUsageType refers to whether it was a taxi or a driving school vehicle (more relevant in Europe). That list doesn't really have anything to do with cars as Americans understand them. There's no provision for a privately-owned vehicle, nor the categories that are fundamental in this market. Americans generally drive themselves around in their own cars, and because America is vast and has a much lower population density than Europe, there are lots of Americans living in wide open spaces, in rural communities, farming and ranching, driving pickup trucks and SUVs. Pickup trucks are the most important and profitable vehicle category for all three US auto companies, and SUVs and minivans are very important as well. Americans' relationship with cars is heavily influenced by having families with several children and needing to drive them around to do cool stuff. The extension takes no account of trucks, SUVs, or minivans, or private ownership as such.

Also, there's a speed property, referring to max possible speed. That won't make any sense to Americans, though I guess no one has to use it. All modern cars and trucks can go faster than 100 mph – nobody cares about the max, since it's faster than we'll ever need, and it's not something you'll ever see a US company put in an ad. Max speed as a property probably makes more sense for something like a motorized bicycle, or an electric city car. That's fine, but the properties US companies would want are missing, like safety and crash ratings (in stars), and quality or dependability awards from places like _Consumer Reports_ and _JD Power and Associates_. Safety ratings and quality ratings are just fundamental to how millions of Americans buy cars – and again, _Americans buy cars_.

Beyond that, there are some big missing elements (well, properties).

- Electric cars as a type – it's odd to roll out a schema in 2017, which might gain wide adoption around 2020 or so, and not explicitly account for electric cars... That threw me.
- Hybrids as a type – again, 2017 yada yada. It's odd to not see hybrids in here.
- Warranties – there's no mention of them.
- Leases. When I said American buy cars, I should've said Americans buy _or lease_ cars. Leases are hugely popular in America, millions of people out there leasing cars and shopping for good lease terms. The US auto sector is unlikely to use a schema that doesn't support this.
- Financing. This is arguably the biggest missing piece. The modal way that Americans buy cars is by financing, a low-interest 5-ish year loan. That might be the single most communicated and marketed information in the US auto market – carmakers offer financing. I think most Americans finance their cars through the automaker instead of third-party banks or credit unions, because the rates are so low.

Basically, we're missing all the stuff that's important when people have their own cars and have to make decisions about cars every few years – warranties, lease terms, finance terms, quality ratings, crash test ratings, etc.

This extension will need to be rewritten if we want to include the US. As it stands, it's incapable of expressing what would be considered critical information here.

**Comment by vholland** [comment]
I don't think the situation is as dire as suggested. While some of the terminology is different than used in the US, the concepts are the same, so towing capacity can be modeled regardless of the property name. Also, these terms need not be reflected in the UI presented to a user.

Part of the issue is much like http://schema.org/Book, we are using http://schema.org/Vehicle to describe both the abstract notion of a type of vehicle and specific instances of vehicles. For example, properties like http://schema.org/mileageFromOdometer are for an instance of a car, while http://schema.org/fuelType is more for the make/model of the car. Perhaps some of the questions could be answered with more examples for types like http://schema.org/Car.

Some of the other things you mention are discussed in other types. For example, leases are covered in http://schema.org/Offer, which also has a way to discuss the warranty using http://schema.org/warranty. Ratings, particularly official ratings like crash star ratings, are something we are expanding with http://schema.org/starRating. We could certainly expand that to be used with http://schema.org/Product as well.

Sewing together the different types and parts of the schema to express these larger concepts is tricky and a place where we could use better documentation and examples.

**Comment by trypuz** [comment]
Thanks @JoeArizona. Please note that http://auto.schema.org/bodyType property indicates the design and body style of the vehicle. 

@JoeArizona could you explain why you think schema.org should have a privately-owned vehicle as a type?

I'm not sure we need two more vehicle/car types, namely Electric vehicle and Hybrid vehicle. We do already have http://auto.schema.org/engineType property that specifies the type of engine or engines powering the vehicle.

I do agree with @JoeArizona that the financial aspects are important. We shall see how new financial properties that are in pending now (see e.g. http://pending.webschemas.org/loanType or http://pending.webschemas.org/loanTerm) can be reused in the context of car financing.

I do agree with @vholland that http://schema.org/starRating should have http://schema.org/Product in its domain.

**Comment by JoeUX** [comment]
Sorry for the delay. I think the properties you mentioned are open-ended
text fields, which makes them much less useful. What I mean is that there
is no category called "PickupTruck", "Coupe", etc. (Maybe I missed them.)
We see the same problem for engine type and transmission type. These need
to have explicit, standard choices, not an open-ended text field. Engines
types would be things 4-cylinder,V6, V8, while transmission type would be
manual, automatic, automanual, double-clutch, etc. with the number of
speeds/gears as a separate facet. Leaving this stuff to open-ended text
will just lead to needless variance in how sites populate these fields and
make search less efficients.

And Schema.org is riddled with non-standard unit abbreviations for some
reason. like HM for miles per hour and KGM from kilograms. The standard
abbreviations are mph and km, so I'm not sure why mew abbreviations are
needed.

And as I mentioned before, safety and crash test ratings are hugely
important in the US, as are emissions categories like Low Emissions Vehicle
(LEV), Ultra Low Emissions Vehicle (ULEV).

Cheers,

Joe
